public/app/plugins/datasource/cloudwatch/tracking.test.ts

```diff
@@ -0,0 +1,767 @@
+                  refId: 'A',
+                  region: 'default',
+                  sqlExpression: '',
+                  statistic: 'Average',
+                },
+                {
+                  alias: '',
+                  datasource: {
+                    type: 'cloudwatch',
+                    uid: 'abc',
+                  },
+                  dimensions: {
+                    InstanceId: '*',
+                  },
+                  expression: '',
+                  hide: false,
+                  id: 'a',
+                  matchExact: true,
+                  metricEditorMode: 0,
+                  metricName: 'CPUUtilization',
+                  metricQueryType: 0,
+                  namespace: 'AWS/EC2',
+                  period: '',
+                  queryMode: 'Metrics',
+                  refId: 'B',
+                  region: 'default',
+                  sqlExpression: '',
+                  statistic: 'Average',
+                },
+                {
+                  alias: '',
+                  datasource: {
+                    type: 'cloudwatch',
+                    uid: 'abc',
+                  },
+                  dimensions: {
+                    InstanceId: 'i-123',
+                  },
+                  expression: '',
+                  id: 'a',
+                  matchExact: true,
+                  metricEditorMode: 0,
+                  metricName: 'CPUUtilization',
+                  metricQueryType: 0,
+                  namespace: 'AWS/EC2',
+                  period: '',
+                  queryMode: 'Metrics',
+                  refId: 'A',
+                  region: 'default',
+                  sqlExpression: '',
+                  statistic: 'Average',
+                },
+                {
+                  alias: '',
+                  datasource: {
+                    type: 'cloudwatch',
+                    uid: 'abc',
+                  },
+                  dimensions: {
+                    InstanceId: '*',
+                  },
+                  expression: 'a / ',
+                  hide: false,
+                  id: '',
+                  matchExact: true,
+                  metricEditorMode: 1,
+                  metricName: 'CPUUtilization',
+                  metricQueryType: 0,
+                  namespace: 'AWS/EC2',
+                  period: '',
+                  queryMode: 'Metrics',
+                  refId: 'B',
+                  region: 'default',
+                  sqlExpression: '',
+                  statistic: 'Average',
+                },
+              ] as CloudWatchQuery[],
+            },
+          })
+        );
+      }),
+    }),
+  };
+});
+
+describe('onDashboardLoadedHandler', () => {
+  it('should report a `grafana_ds_cloudwatch_dashboard_loaded` interaction ', () => {
+    expect(reportInteraction).toHaveBeenCalledWith('grafana_ds_cloudwatch_dashboard_loaded', {
+      dashboard_id: 'dashboard123',
+      grafana_version: 'v9.0.0',
```

Comment:
With the event-bus solution, we now have access to the plugin version, which might be useful to add `ds_version: pluginJson.info.version` ?